src/domain/entities/atlassian-design.ts

```diff
@@ -22,7 +20,7 @@ export class AtlassianAssociation {
 	) {}
 
 	static createDesignIssueAssociation(issueAri: string): AtlassianAssociation {
-		return new AtlassianAssociation(JIRA_ISSUE_ATI, [issueAri]);
+		return new AtlassianAssociation('ati:cloud:jira:issue', [issueAri]);
```

Comment:
**note:** Remove the constant and its usages to avoid this concept being leaked to other modules. The whole concept of `AtlassianAssociation` will be removed shortly due to the simplification on Atlassian side (work in progress).